Realm/RealmSwift/Migration.swift

```diff
@@ -54,23 +54,26 @@ public class Migration {
 
     private var rlmMigration: RLMMigration
 
-    init(_ rlmMigration: RLMMigration) {
+    internal init(_ rlmMigration: RLMMigration) {
         self.rlmMigration = rlmMigration
     }
 }
 
 public class MigrationObject : Object {
-    subscript(key: String) -> AnyObject? {
-        if (self.objectSchema[key].type == RLMPropertyType.Array) {
-            return listProperties[key]
+    public override subscript(key: String) -> AnyObject? {
+        get {
+            if (objectSchema[key].type == .Array) {
```

Comment:
Please revert this change to subscript. The setter does not work for list properties. This should be handled in a different pr.